databases/week1/homework.sql

```diff
@@ -0,0 +1,75 @@
+-- ques1 : Find out how many tasks are in the task table
+select count(id)
+from task;
+
+
+-- ques2 : Find out how many tasks in the task table do not have a valid due date
+
+select count(title)
+from task
+where due_date is null;
+
+
+-- ques3 : Find all the tasks that are marked as done
+
+select count(title)
+from task
+where status_id = 3 ;
+
+
+-- ques4 : Find all the tasks that are not marked as done
+
+select count(title)
+from task
+where status_id = '1' or status_id = '2';
```

Comment:
there is a simpler way to check if the status is not "done", which would make your life much easier if there were 20 different status names, do you know which way it is?

Reply:
Yes i got it i could have done it like : where status_id != 3 ;
it will send me everything except 3(which is done).

Reply:
yes! :)

Reply:
just update this query and then i'll approve the PR :)